docs/x509/ocsp.rst

```diff
@@ -690,3 +707,71 @@ Interfaces
 
         Encode the X.509 ``Name`` of the certificate whose private key signed
         the response.
+
+.. class:: OCSPResponseIterator
```

Comment:
So, this isn't actually documented as being an iterator, this is documenting a `SingleResponse` -- there's no `__next__` on this class, as documented!